src/Button/index.tsx

```diff
@@ -48,7 +50,7 @@ const Button: FunctionComponent<Props> = ({
 
   if (href) {
     return (
-      <a href={href} className={btnClassName} {...rest}>
+      <a href={href} className={clsx(btnClassName, disableUnderline && styles.disableUnderline)} {...rest}>
```

Comment:
I'm not sure I would add this as a prop since it seems like a one off case. I would just use the `className` prop to make this css change